### PR TITLE
[webmessaging] Fixing test

### PR DESCRIPTION
**Pull request by Gabisampaio**
This test was passing but no assert was being ran. Fixing the test so
that assert_equals is called.
Issue: https://github.com/web-platform-tests/wpt/issues/29536?fbclid=IwAR0LIrp8Qv5GsFp_0ZA8VFpEfhTmr5cxbt3WE-O1o_-eWFE1TNAOnFZLLxA

### CODE REVIEW

File: workers/constructors/SharedWorker/interface-objects.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
   worker.port.start();
   worker.port.postMessage(supported);
   worker.port.onmessage = t.step_func_done(e => {
-    assert_equals(e.data, '');
+    assert_equals(e.data, 'These were missing: SharedWorker');
```
Comment (mkruisselbrink) [inline workers/constructors/SharedWorker/interface-objects.html:17]: This change should probably be a in a separate PR? Also rather than changing the assert here, I think removing SharedWorker from the expected list on line 7 would make more sense.

Comment (mkruisselbrink) [reply workers/constructors/SharedWorker/interface-objects.html:17]: (while you're at it you might also want to remove ApplicationCache from that list, as that API isn't in the spec anymore)

Comment (Gabisampaio) [reply workers/constructors/SharedWorker/interface-objects.html]: @mkruisselbrink Thanks for the great feedback! I put the change in a [separate pull request](https://github.com/web-platform-tests/wpt/pull/29988/files) as you advised and applied your suggestions.

File: webmessaging/with-options/one-arg.html [hunk 2]
```diff
@@ -4,11 +4,10 @@
...
 <div id="log"></div>
 <script>
-test(function() {
+async_test(t => {
   postMessage('');
-  onmessage = this.step_func(function(e) {
-    assert_equals(e.data, '');
-    this.done();
+  onmessage = t.step_func_done(e => {
+  	assert_equals(e.data, '');
```
Comment (annevk) [inline webmessaging/with-options/one-arg.html:10]: Please remove the tab character here:
```suggestion
    assert_equals(e.data, '');
```

File: webmessaging/with-options/one-arg.html [hunk 3]
```diff
@@ -4,11 +4,10 @@
...
   });
-});
+ });
```
Comment (annevk) [inline webmessaging/with-options/one-arg.html:12]: This indentation seems to be in error unless I'm missing something.
```suggestion
});
```

**Review by annevk — COMMENTED** [review]
Thanks, just two small nits.

**Comment by Gabisampaio** [comment]
@annevk Thanks again for your feedback! I just applied the suggestions :-)

**Review by annevk — APPROVED** [review]
